### PR TITLE
Update buisness-logic

### PR DESCRIPTION
**Pull request by zairigimad**
Replace all folders names with symfony3-project instead of  symfony2-project because symfony 2 and 3 don't have the same structure of folders .

### CODE REVIEW

File: best_practices/business-logic.rst [hunk 1]
```diff
@@ -35,7 +35,7 @@ and put things there:
 
 .. code-block:: text
 
-    symfony2-project/
+    symfony3-project/
```
Comment (snoek09) [inline best_practices/business-logic.rst]: Why not just call it `symfony-project`?

Comment (zairigimad) [inline best_practices/business-logic.rst]: Yes it will be generic !

**Comment by snoek09** [comment]
Thanks @zairigimad :+1: 

Status: reviewed.

**Comment by javiereguiluz** [comment]
👍

**Comment by wouterj** [comment]
👍

**Comment by wouterj** [comment]
Thanks Zairig, nice improvement! I've merged it into the 2.7 version of the docs (the oldest version we still maintain) and I'll merge it into the newer versions from there.

**Comment by zairigimad** [comment]
@wouterj Welcome , i'm happy to contribute in my favorite framework :+1:

**Pull request closed without merging**